src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -207,11 +208,22 @@ private void checkWhiteListLabels() {
 
     private void checkSkipBuild() {
         synchronized (this) {
-            String skipBuildPhrase = helper.checkSkipBuild(this.pr);
+            String skipBuildPhrase = helper.checkSkipBuildPhrase(this.pr);
             if (!StringUtils.isEmpty(skipBuildPhrase)) {
                 logger.log(Level.INFO,
-                           "Pull request commented with {0} skipBuildPhrase. Hence skipping the build.",
-                           skipBuildPhrase);
+                        "Pull request commented with {0} skipBuildPhrase. Hence skipping the build.",
+                        skipBuildPhrase);
+                shouldRun = false;
+                return;
+            }
+            if (commitAuthor == null){
+                return;
```

Comment:
How about logging a `FINE` message here? I honestly don't know if this condition happens but it'd be nice to have logging if it does come up.